pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -222,5 +228,39 @@ func ValidateVirtualMachineSpec(field *k8sfield.Path, spec *v1.VirtualMachineSpe
 		}
 	}
 
+	return causes
+}
+
+func validateVMStartAllowed(vm *v1.VirtualMachine) []metav1.StatusCause {
```

Comment:
Not sure, maybe `preventStartIfRenameInProgress`?

Reply:
I think I should prevent ANY kind of change to a vm (not only starting it) if a renaming process is in progress, to prevent any updates or patches to the old vm that would not carry over to the new one, what do you think?